archlinux/PKGBUILD

```diff
@@ -4,29 +4,27 @@
 # then please put 'unknown'.
 
 # Maintainer: Olivier Medoc <o_medoc@yahoo.fr>
-pkgname=qubes-vm-gui
+pkgname=(qubes-vm-gui qubes-vm-pulseaudio)
```

Comment:
How upgrades are handled in such case?

Reply:
You are right, upgrades are not handled in this case and lot of users will complain.

I need to think if it should be kept optional, create a dependency, or use a meta package as you suggested.

I try to not use dependencies because of pulseaudio / xorg upgrade issues. I would like to keep the possibility to uninstal only the pulseaudio component without breaking the package manager.

Reply:
Ok, since documentation now (will) include mention of extra package for sound support, IMO it is ok to simply split the package here.